src/styles/blocks/card.scss

```diff
@@ -0,0 +1,112 @@
+@import '/src/styles/blocks/variables';
+
+main {
+  display: grid;
+  gap: 46px 48px;
+  padding: 50px 40px;
+  grid-template-columns: repeat(1, $card-size);
+  place-items: center;
+  place-content: center;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, $card-size);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, $card-size);
+  }
+
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, $card-size);
+  }
+}
+
+.card {
+  width: $card-size;
+  box-sizing: border-box;
+  background-color: rgba(255, 255, 255, 1);
+  border: 1px solid rgba(243, 243, 243, 1);
+  font-family: $main-font-family;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  border-radius: 5px;
+  transition: $transition-time;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__photo {
+    background-image: url(../images/imac.jpeg);
+    background-position: center;
+    background-size: contain;
+    background-size: 100% 100%;
```

Comment:
The `background-size` property is set twice for `.card__photo`. Consider removing one of these declarations to avoid redundancy. If you want the image to cover the entire area, use `background-size: cover;` instead.